common/persistence/sql/executions.sql

```diff
@@ -0,0 +1,68 @@
+CREATE TABLE IF NOT EXISTS transfer_tasks(
```

Comment:
Put all the sql setups in one file, call it initial_tables.sql maybe. That's how it will be used by the upgrade framework later on.
Also put it in a sub-dir.